projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -343,6 +343,12 @@ function wpcom_launchpad_get_task_definitions() {
 			},
 			'is_complete_callback' => 'wpcom_is_task_option_completed',
 			'is_visible_callback'  => 'wpcom_is_enable_subscribers_modal_visible',
+			'get_calypso_path'     => function ( $task, $default, $data ) {
+				if ( ( new Automattic\Jetpack\Status\Host() )->is_atomic_platform() ) {
+					return admin_url( '/admin.php?page=jetpack#/discussion' );
```

Comment:
Nit: I think it's more common to drop the leading `/`, but this still seems to work.
```suggestion
					return admin_url( 'admin.php?page=jetpack#/discussion' );
```